src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+import {propTypes, defaultProps} from '../anchorForCommentsOnlyPropTypes';
+
+/*
+ * This is a default anchor component for regular links.
+ */
+const BaseAnchorForCommentsOnly = ({
+    href,
+    rel,
+    target,
+    children,
+    style,
+    ...props
+}) => (
+    <Text
+        style={StyleSheet.flatten(style)}
+        accessibilityRole="link"
+        href={href}
+        rel={rel}
+        target={target}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+    >
+        {children}
+    </Text>
+);
+
+BaseAnchorForCommentsOnly.propTypes = propTypes;
+BaseAnchorForCommentsOnly.defaultProps = defaultProps;
+BaseAnchorForCommentsOnly.displayName = 'BaseAnchorForCommentsOnly';
+
+export default BaseAnchorForCommentsOnly;
```

Comment:
Perhaps a dumb question, but why do we have this file and the `index.native.js` file? They both export the same thing (`BaseAnchorForCommentsOnly`). When and how does one get used instead of the other?

Reply:
`index.js` is the web/desktop implementation, while `index.native.js` is the ios/android implementation. These are only separated when necessary. There's some more information [here](https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions)